src/Util.js

```diff
@@ -49,7 +49,8 @@ export function formatStyle (style, metadata, styleUrl) {
       style.sources.esri.url + metadata.tiles[0]
     ],
     description: metadata.description,
-    name: metadata.name
+    name: metadata.name,
+    maxzoom: metadata.maxzoom ? metadata.maxzoom : 22
```

Comment:
might make more sense to use the same value as our hosted vector tiles as the default.

https://basemaps.arcgis.com/v1/arcgis/rest/services/World_Basemap/VectorTileServer?f=json

```js
"lods": [
  // ...
  {
    "level": 18,
    "resolution": 0.29858214164761665,
    "scale": 1128.497176
  }
]
```